src/main/java/util/XMLParser.java

```diff
@@ -0,0 +1,62 @@
+package util;
```

Comment:
What is the purpose of having XMLParser classes in 2 different packages?

Reply:
Actually, I though I deleted this one.

Reply:
Than good